src/pages/users/Tabs/RowAccessTab.js

```diff
@@ -180,6 +195,19 @@ const RowAccessTab = ({ maxAccess, labels, storeRecordId }) => {
     >
       <VertLayout>
         <Fixed>
+          <Grid item xs={3} sx={{ mb: 3 }}>
```

Comment:
![image](https://github.com/user-attachments/assets/323aee42-2110-4af0-bf8e-9f5a14dbc52c)
please add the search next to the combo not above, we need maximum space for the below grid to show data